doc/btrfs-multiple-devices.md

```diff
@@ -0,0 +1,101 @@
+
+Multiple Devices Btrfs
+======================
+
+
+Mounting Degraded Btrfs
+-----------------------
+
+Mounting a degraded btrfs requires the 'degraded' mount option.
+
+Using the 'degraded' mount option in /etc/fstab by default was not recommended
+by btrfs experts (on IRC).
+
+To be decided: What should be done during probing with degraded btrfses?
+
+The 'degraded' option is a nop if no devices are missing.
+
+
+Changing RAID Levels
+--------------------
+
+Changing the RAID levels required running btrfs balance job.
+
+  btrfs balance start -dconvert=raid1,soft -mconvert=raid1,soft /test
+
+Changing the RAID levels can require copying huge amounts of data so is
+potentially very time consuming. Usually such operations are not supported by
+libstorage-ng.
+
+As a benefit the balance can be started in the background. But this fails if
+there is already a balance job running (without an error message from the
+btrfs command). The system does not provide a way to queue balance jobs and it
+should not be the task of libstorage-ng to implement such a way.
+
+The balance job can also fail due to missing space. When run in the background
+reporting that to the user is not possible (the balance job starts and simply
+fails later).
```

Comment:
no even email or message to journalctl?

Reply:
There are error messages visible with dmesg. But since YaST would likely not run at that time  this does not help much.

Reply:
well, we can at least hint user, that it is asynchronous operation and that he should check dmesg for potential problems.